pkg/webui/console/views/device-general-settings/network-server-form/index.js

```diff
@@ -51,14 +54,22 @@ import {
 
 import validationSchema from './validation-schema'
 
+const m = defineMessages({
+  resetTitle: 'Reset to factory defaults',
+  resetSuccess: 'End device reset',
+  resetFailure: 'There was an error resetting the end device',
+  modalMessage:
+    'Are you sure you want to reset session context and MAC state of this end device? For OTAA all data is wiped and end device must rejoin, for ABP session keys, device address and downlink queue are preserved, while MAC state is reset.',
```

Comment:
```suggestion
    'Are you sure you want to reset the session context and MAC state of this end device?<br /><br />This will have the following consequences:<ul><li>For <DocLink>OTAA</DocLink>-activated end devices <b>all session and MAC data is wiped and the end device MUST rejoin</b></li><li>For <DocLink>ABP</DocLink>-activated end devices, session keys, device address and downlink queue are preserved, while <b>the MAC state is reset</b></li></ul>',
```

Let's be as clear as possible here. @htdvisser @adriansmares please review this sentence and let us know if anything else should be brought to the users attention when resetting.

`DocLink` should point to `/devices/abp-vs-otaa`

Reply:
Text LGTM, but `Over The Air Activation (OTAA)-activated` is extremely verbose, also we don't give the full name for ABP.

Maybe show `Over The Air Activation` / `Activation By Personalization` on hover ?

Reply:
I've updated this to include a link to the documentation about activation modes and then use the abbreviation only.